app/webroot/js/sentences.edit_in_place.js

```diff
@@ -53,9 +60,23 @@ $(document).ready(function() {
             cssclass  : 'editInPlaceForm',
             onblur    : 'ignore'
         }).bind('edit_sentence', function(e) {
+            $(this).find('textarea').keyup(function(event) {
+                var submitBtn = $(this).parent().find('button[type=submit]');
+                if($(this).val().trim().length == 0) {
+                    submitBtn.prop('disabled', true);
+                    submitBtn.css("background-color","gray");
```

Comment:
It would be better to define the CSS in the CSS files rather than set it in the Javascript.

You can add this in the `elements.css`:
````
.editableSentence button[disabled] {
    background: #ccc;
    border-color: #aaa;
}
````